components/OverflowMenu.js

```diff
@@ -64,8 +64,10 @@ class OverflowMenu extends Component {
   }
 
   getMenuPosition = () => {
-    const menuPosition = this.menuEl.getBoundingClientRect();
-    this.setState({ menuPosition });
+    if (this.menuEl) {
```

Comment:
@chrisdhanaraj ? 

There's probably a good reason for this but I'm not sure exactly why this is always needed. 
My understanding is: since `getMenuPosition()` is used in `componentDidMount()`, then it will have access to `refs` ...but according to this PR, it's being implied that the component attempts to use `refs` before they're defined.